Zero-Gravity-Store-React/src/Pages/HomePage/HomePage.js

```diff
@@ -1,13 +1,23 @@
+import { useState } from "react";
+import {Loader} from "../../Components/Loader/Loader";
 import { Footer } from "../../Components/Footer/Footer";
 import { Navigation } from "../../Components/Navigation/Navigation";
 import { HeroComponents } from "./Components/HeroComponents";
 import "./HomePage.css";
+import { BarLoader, RingLoader, RiseLoader } from "react-spinners";
 export function HomePage() {
+  const [loading, setLoading] = useState(true);
+  setTimeout(()=>{
+    setLoading(false)
+  },4000);
```

Comment:
Why set timeout have been used here ?.

Reply:
To set time out for Loader as home page has no API calls in mine